### ISSUE TITLE
Pico ducky not injecting keystrokes in Linux

### ISSUE THREAD

**Issue by fedepau** [dbisu/pico-ducky#207]
Hi,

I just created a pico ducky with a Raspberry Pi Pico W on Arch Linux. I followed the instructions provided to the letter (both with CircuitPython 8.0.0 and 8.2.6, the latest release). Upon transferring the .uf2 file to the Pico, it behaves as expected and reboots showing up as CIRCUITPY.
I proceeded with the instructions and also changed my keyboard layout to Italian to match my keyboard, then created this sample payload with Payload Studio, which spawns a terminal (keybind Win+Enter on my system) and types some dummy text:

```
DELAY 3000
GUI ENTER
DELAY 500
STRING gotcha :3
```

If I understand the instructions, after copying the payload to the root of my pico it should reboot into "attack" mode and immediately start injecting keystrokes (since I didn't ground GP0). However, this does not happen: the device remains mounted as a mass storage device, and does not inject any keystroke. If I disconnect and re-connect it, it successfully "hides" (= does not show up as USB mass storage), but still does not inject any keystroke. 
Here's `minicom -b 115200 -o -D /dev/ttyACM0`:
```
Auto-reload is off.
code.py output:
0;🐍Wi-Fi: No IP | code.py | 8.0.0progStatus False
Finding payload
Running  payload.dd
Traceback (most recent call last):
  File "code.py", line 59, in <module>
  File "duckyinpython.py", line 143, in runScript
UnicodeError: 
0;🐍Wi-Fi: No IP | 143@duckyinpython.py UnicodeError | 8.0.0
Code done running.
```
By inspecting the files, it seems that I correctly flashed pico-ducky on my pico, but my payload either isn't found, or is formatted in such a way that pico-ducky is unable to run it. Any hint on how to solve this?

**Comment by fedepau** [comment]
Ok, silly me. I managed to fix the issue myself. As the error suggests, it was actually a string formatting issue. Apparently, PayloadStudio is not needed in Linux, I just manually edited `payload.dd` with its content in ducky-script (UTF-8 encoding) and it worked flawlessly. Still a minor issue tho:
If I ground 'GP0` and connect my ducky to my computer, it still won't be recognized as a USB mass storage. How can I do so, in order to edit/adding payloads?
